File: 01FlutterApps/front_dart_flutter/lib/03StuDocMngmnt/3bankBasic/Kn03D003BnkBean.dart
class Kn03D003BnkBean {
  final String bankId;
  final String bankName;
  final int delFlg;

  Kn03D003BnkBean( {
    required this.bankId, 
    required this.bankName, 
    required this.delFlg,
    });

  factory Kn03D003BnkBean.fromJson(Map<String, dynamic> json) {
    return Kn03D003BnkBean(
      bankId: json['bankId'],
      bankName: json['bankName'],
      delFlg: json['delFlg'] ?? 0,
    );
  }
}

File: 01FlutterApps/middle_layer_springboot/knpiano_java/src/main/java/com/liu/springboot04web/bean/Kn03D003BnkBean.java
package com.liu.springboot04web.bean;

import java.util.Date;

public class Kn03D003BnkBean implements KnPianoBean {
    private String bankId;
    private String bankName;
    private Integer delFlg;

    private Date createDate;
    private Date updateDate;

    public String getBankId() {
        return bankId;
    }

    public void setBankId(String bankId) {
        this.bankId = bankId;
    }

    public String getBankName() {
        return bankName;
    }

    public void setBankName(String bankName) {
        this.bankName = bankName;
    }

    public Integer getDelFlg() {
        return delFlg;
    }

    public void setDelFlg(Integer delFlg) {
        this.delFlg = delFlg;
    }

    public Date getCreateDate() {
        return createDate;
    }

    public void setCreateDate(Date createDate) {
        this.createDate = createDate;
    }

    public Date getUpdateDate() {
        return updateDate;
    }

    public void setUpdateDate(Date updateDate) {
        this.updateDate = updateDate;
    }
}
